### ISSUE TITLE
Support for Sampling/1.0

### ISSUE THREAD

**Issue by htulipe** [highsource/ogc-schemas#96]
Hi, I would like to be able to read Sampling 1.0 documents. Do you plan on supporting it soon or can I contribute it? If you are open to contribution, could you explain the process to use.

Thanks.

**Comment by highsource** [comment]
Hi,

I'd appreciate a contribution, the best would be a pull request.
You could do it analogous to the Sampling/2.0:

https://github.com/highsource/ogc-schemas/tree/master/sampling/2.0
https://github.com/highsource/ogc-schemas/blob/master/scripts/tests/Sampling/2.0

I'm currently under a huge stress and don't quite come to my open source
projects, so a pull request would be great.

Best wishes,
Alexey

**Comment by htulipe** [comment]
Ok i forked your repo. Is it normal that I have a failing test? When I try to build the "scripts" Maven module, I get:

    Testing [....\ogc-schemas\scripts\tests\WPS\2.0/describeProcess_request.xml].
    Error: Element [{http://www.opengis.net/wps/2.0.0}wps:DescribeProcess] is not known in this context and property does not allow DOM.

**Comment by htulipe** [comment]
Oddly enough the following commands report changed files :

    mvn clean install
    git status

There are lot of them, most of them are xsd and are located in  `schemas/src/main/resources` or `schemas/src/main/resources-original`

**Comment by highsource** [comment]
This test currently fails on my machine as well, see #100.

Changed files are OK. They're checked in... well, let's say, for historical reasons.
The build process downloads schemas from schemas.opengis.net but then also patches them in a few places.
I've decided to check the schemas in, this makes it a bit easier to track changes in schemas.
They are actually build artifacts and should not be really checked in, but I've decided that this is a bit unique situation and thought it'll be more practical to commit them.
